kotishell/src/lists/lists_b.c

```diff
@@ -0,0 +1,39 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   lists_b.c                                          :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: elenavoronin <elnvoronin@gmail.com>          +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2023/06/30 15:31:04 by evoronin      #+#    #+#                 */
+/*   Updated: 2023/07/12 11:54:09 by evoronin      ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "lists.h"
+#include "tokenize.h"
+
+void	li_list_free(t_li_node *list)
```

Comment:
need func parameter to properly free data

Reply:
moved to the tokenize module and made it token specific